no-computer.md

```diff
@@ -57,13 +98,27 @@ spent directly engaging in the **language** aspects of this project (e.g.,
 making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
+Language, which involves defining vocabulary and rules, takes 90 percent of the
```

Comment:
This actually sounds like a really interesting game and one that could turn into a DSL!